examples/hello_world.py

```diff
@@ -0,0 +1,25 @@
+
+class App():
+    def __init__(self, scope):
+        self.scope = scope
+
+    async def __call__(self, receive, send):
+        await send({
+            'type': 'http.response.start',
+            'status': 200,
+            'headers': [
+                [b'content-type', b'text/plain'],
+            ],
+        })
+        await send({
+            'type': 'http.response.body',
+            'body': b'Hello, world!',
+        })
+
+
+if __name__ == '__main__':
+    # The above can be used using the ASGI-compatible cli,
+    # or we can run in programatically:
+    import trio
+    from trio_web import serve_tcp as serve_http  # todo: why is it called serve_tcp
```

Comment:
...and why is it called trio_web instead of trio_asgi? Yeah, I know! :)

Reply:
That too yes :D